### PR TITLE
DEVOPS-43: Adding Production Jenkinsfile 

### PR DESCRIPTION
**Pull request by lenardw1**
The production Jenkinsfile will pull down master, tag it with the latest release, update/store docker images, and deploy to production

### CODE REVIEW

File: devops-accounts-service/Dockerfile [hunk 1]
```diff
@@ -9,9 +9,11 @@ RUN apt-get update && apt-get -y install python curl unzip && cd /tmp && \
     rm awscli-bundle.zip && rm -rf awscli-bundle \
 	&& rm -rf /var/lib/apt/lists/*
 ARG WAR_FILE
+ARG DB_USER
+ARG DB_PASS
 ENV PROFILE ${PROFILE}
 ENV RELEASE_VERSION ${RELEASE_VERSION}
 ENV SECRETS_BUCKET_NAME ${SECRETS_BUCKET_NAME}
```
Comment (RussianPrussian) [inline devops-accounts-service/Dockerfile]: get rid of this line. Don't need "SECRETS_BUCKET_NAME" anymore

File: Jenkinsfile-prod [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    }
+  }
+  post {
+       success {
+           sendToSlack("Production Deployment Completed", "success")
+       }
+       failure {
+            sendToSlack("Production Deployment Failed", "failure")
+            
+            emailext (subject: '$PROJECT_NAME - Release # $releaseNumber - $BUILD_STATUS!', body: 'The buid failed and you might be the culprit.<br/><br/>Please view the build results at $BUILD_URL.<br/><br>Thanks,<br><br>Mr. Jenkins', mimeType: 'text/html', from: 'noreply@jenkinspsi.com', replyTo: 'noreply@jenkinspsi.com', recipientProviders: [[$class: 'CulpritsRecipientProvider']])
```
[the remark this answers is not in the document]
Comment (lenardw1) [reply Jenkinsfile-prod]: Agreed

File: Jenkinsfile-prod [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+      }
+    }
+
+    stage('Build UI Docker Image') {
+      steps {
+        sh 'git checkout master'
+
+        withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: '5b27964b-fd8d-4537-a373-646b0906a386',
+        accessKeyVariable: 'AWS_ACCESS_KEY_ID',  secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+          buildImage("psi-devops-ui-prod","${releaseNumber}",  "devops-ui", "--build-arg env=\"devtest\"")
```
[the remark this answers is not in the document]
Comment (lenardw1) [reply Jenkinsfile-prod]: Yea your right. Good catch

File: Jenkinsfile-prod [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    }
+  }
+  post {
+       success {
+           sendToSlack("Production Deployment Completed", "success")
+       }
+       failure {
+            sendToSlack("Production Deployment Failed", "failure")
+            
+            emailext (subject: '$PROJECT_NAME - Release # $releaseNumber - $BUILD_STATUS!', body: 'Production deployment failed.<br/><br/>Please view the results at $BUILD_URL.<br/><br>Thanks,<br><br>Mr. Jenkins', mimeType: 'text/html', from: 'noreply@jenkinspsi.com', replyTo: 'noreply@jenkinspsi.com', recipientProviders: [[$class: 'CulpritsRecipientProvider']])
```
Comment (RussianPrussian) [inline Jenkinsfile-prod]: Wonder if we should refine the recipient. I'm still going to approve this now. But yeah. Something to think about.

**Pull request merged**